Yoong/1110_programmers_단속카메라.py

```diff
@@ -0,0 +1,20 @@
+# https://school.programmers.co.kr/learn/courses/30/lessons/42884
+
+def solution(routes):
+    B = sorted(routes, key = lambda x: x[1], reverse=False) # 나간 지점을 기준으로 오름차순으로 정렬
+
+    k = 0
+    cnt = 1                                                 # 처음 나간 차량의 나간 지점에 하나 설치 먼저함.
+    for i in range(1,len(B)):                               # 두번째 차량부터 마지막 번째 차량까지 반복문 실행
+        if B[i][0] > B[k][1]:                               # 다음 차량이 들어온 지점과 카메라가 설치되어 있는 지점을 비교
+            cnt += 1                                        # 카메라가 설치된 지점보다 다음 차량의 들어온 시간이 늦으면 카메라 설치
+            k = i                                           # 카메라 지점을 다음 차량의 나간 지점으로 업데이트함.
+    #return B, cnt
+    return cnt
```

Comment:
```suggestion
def solution(routes):
    routes.sort(routes, key = lambda x: x[1], reverse=False) # 나간 지점을 기준으로 오름차순으로 정렬

    k = 0
    cnt = 1                                                 # 처음 나간 차량의 나간 지점에 하나 설치 먼저함.
    for i in range(1,len(routes)):                               # 두번째 차량부터 마지막 번째 차량까지 반복문 실행
        if routes[i][0] > routes[k][1]:                               # 다음 차량이 들어온 지점과 카메라가 설치되어 있는 지점을 비교
            cnt += 1                                        # 카메라가 설치된 지점보다 다음 차량의 들어온 시간이 늦으면 카메라 설치
            k = i                                           # 카메라 지점을 다음 차량의 나간 지점으로 업데이트함.
    #return routes, cnt
    return cnt
```

이런식으로 리팩토링 해보는건 어떨까요??
B라는 변수 이름이 남들이 판단하기에 좋지 않은 네이밍이라고 판단되고 새롭게 변수에 할당되면서 불필요한 추가 메모리 공간이 사용되는 거 같습니다. 의견일 뿐입니당 :)

Reply:
네 그 부분에 대해서 동의합니다 
의견 감사합니다:)